tests/pyopencl/23_bandwidth.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
should we remove this? We should let xbutil decide which pyhton to run

Reply:
shebang should not apply to xbutil? This is only relevant if someone attempts to directly run the script like `./22_bandwidth.py`